RELEASE_NOTES.rst

```diff
@@ -1,5 +1,57 @@
-coala 0.9.0
-===========
+coala 0.9.0 - GlobalBear
+========================
+
+::
```

Comment:
not sure about rst syntax in general, but this is messed up when viewed on GitHub.
Needs a new line after `::`?